### ISSUE TITLE
tf-small accuracy

### ISSUE THREAD

**Issue by happygirlzt** [eth-sri/TFix#18]
Hi there, thank you very much for open sourcing and providing a very detailed README. :)
I tried to replicate the result, and I experimented with tf-small without changing anything. The last line of `first_accs.txt` I got should indicate the averaged accuracy, which is `"average": 0.4435292415316816`. While in the paper, the reported accuracy was `39.2`. If I was not wrong, according to the current `tfix_testing.py`, `first_accs.txt` should record the accuracy in the exact match in the clean test. Basically, the accuracy I got with tf-small was 12.8% better than the original result.
So I was wondering why this case happened. Please correct me if I misunderstood anything.
Thank you in advance.

**Comment by BBerabi** [comment]
hi @happygirlzt ,

Sorry for the late reply. I was on vacation for the last two weeks and glad to hear that you like the README. I invested quite some time into it. :) 

The reported accuracy in the paper is around 39 percent and you have 44 percent. So, I think the difference is around 5 percent but not 12 if I understand you correctly.

In any case, this unfortunately indicates that we have an issue of reproducibility. The Hugging Face library needs "seed fixing" at multiple positions and I think back then I did not know that and only set the seed at one of them. 

Still, I am happy that at least the results are better but not worse than we reported.

I hope this helps!

Best,
Berkay
